rust/agama-cli/src/logs.rs

```diff
@@ -45,6 +50,32 @@ pub async fn run(subcommand: LogsCommands) -> anyhow::Result<()> {
     }
 }
 
+// Whatewer passed in dest formed into an absolute path with archive name
+// if dest is none then a default is returned
+// if dest is directory then a default file name for the archive will be appended
+// TODO: if dest is path with a file name then it is used as is a name for resulting archive
```

Comment:
well, I think this logs can be used in integration tests which now missing many logs ( it use only y2log and then traces from playwright. And for that I ideally need stable and unchanged path. So for me it would be ideal if I can pass complete path like `/tmp/agama.txz` and then point github to that file to upload it as artifact. Will it be possible?

Reply:
it is plan ... see the todo comment

Reply:
@jreidinger just added. Pls check if it fulfills your needs.